recipes/cis/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "1.3.4" %}
+
+package:
+  name: cis
+  version: {{ version }}
+
+source:
+  fn: cis-{{ version }}.tar.gz
+  url: https://github.com/cedadev/cis/archive/{{ version }}.tar.gz
+  sha256: 319f404d9af40baf2f1a2f26c43dd82abee906d1450d9d3db8c48df5a2041ff9
+
+build:
+  number: 0
+  skip: True  # [py3k]
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - matplotlib >=1.2.0
+    - setuptools
+    - netcdf4 >=1.0
+    - numpy
+    - scipy
+    - iris >=1.8.0
+    - pyhdf
+    - psutil
+    - pandas
+
+  run:
+    - python
+    - matplotlib >=1.2.0
+    - netcdf4 >=1.0
+    - numpy
+    - scipy
+    - iris >=1.8.0
+    - pyhdf
+    - psutil
+    - pandas
```

Comment:
It seems that `cis` depends on `pyhamcrest`. Can you add the dependency here instead of only at the test time?